libs/design/src/atoms/form/error-message/error-message.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { DaffErrorMessageComponent } from './error-message.component';
+
+@NgModule({
+  imports: [
+    CommonModule
```

Comment:
I don't know if we need the common module.

Reply:
Removed